data/categories.txt

```diff
@@ -747,7 +747,7 @@ nl:^Buurtwinkel|kruidenier
 fi:^Lähikauppa
 fr:^Supérette
 de:^Gemischtwarenladen
-hu:^Csemegebolt|ábécé
+hu:^Kis élelmiszerbolt|vegyesbolt
```

Comment:
Is the word "Kis" required here? Isn't plain "élelmiszerbolt" enough?

Reply:
​It might be but élelmiszerbolt is a general term which may apply also to big shops, e.g. supermarkets. Kis élelmiszerbolt is more specific and gives a clear explanation that at that POI you will find a relatively small shop. It is a common term also in other OSM applications (OSMand, Vespucci, JOSM, etc.).​